web-ui/src/components/member_selector/member_selector_dialog/MemberSelectorDialog.jsx

```diff
@@ -391,10 +391,10 @@ const MemberSelectorDialog = ({
 
     // Search by member name
     if (nameQuery) {
-      selectable = selectable.filter(member => {
-        const sanitizedQuery = nameQuery.trim().toLowerCase();
```

Comment:
I moved this line out of the filter loop and shortened a variable name to match where I have this same logic in another file below.